sdk/storage/Azure.Storage.Files.Shares/tests/DirectoryClientTests.cs

```diff
@@ -1050,7 +1050,8 @@ public async Task GetFileClient_SpecialCharacters(string directoryName, string f
             ShareFileClient fileFromDirectoryClient = InstrumentClient(directoryClient.GetFileClient(fileName));
             Response<ShareFileInfo> createResponse = await fileFromDirectoryClient.CreateAsync(Constants.KB);
 
-            Uri expectedUri = new Uri($"https://{TestConfigDefault.AccountName}.file.core.windows.net/{test.Share.Name}/{Uri.EscapeDataString(directoryName)}/{Uri.EscapeDataString(fileName)}");
+            string endpointSuffix = Environment.GetEnvironmentVariable("STORAGE_ENDPOINT_SUFFIX") ?? Constants.ConnectionStrings.DefaultEndpointSuffix;
```

Comment:
I don't think we should use env vars directly here. cc @kasobol-msft

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml has service endpoints. these should be used instead. And looks like that template needs work to accommodate these other clouds.